openfe/protocols/openmm_md/plain_md_methods.py

```diff
@@ -0,0 +1,604 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+"""OpenMM MD Protocol --- :mod:`openfe.protocols.openmm_md.plain_md_methods`
+===========================================================================================
+
+This module implements the necessary methodology tools to run an MD
+simulation using OpenMM tools.
+
+"""
+from __future__ import annotations
+
+import logging
+
+from collections import defaultdict
+import gufe
+import openmm
+from openff.units import unit
+from openff.units.openmm import from_openmm, to_openmm
+import openmm.unit as omm_unit
+from typing import Optional
+from openmm import app
+import pathlib
+from typing import Any, Iterable
+import uuid
+import time
+import numpy as np
+import mdtraj
+from mdtraj.reporters import XTCReporter
+from openfe.utils import without_oechem_backend, log_system_probe
+from gufe import (
+    settings, ChemicalSystem, SmallMoleculeComponent,
+    ProteinComponent, SolventComponent
+)
+from openfe.protocols.openmm_md.plain_md_settings import (
+    PlainMDProtocolSettings, SystemSettings,
+    SolvationSettings, OpenMMEngineSettings,
+    IntegratorSettings, SimulationSettingsMD,
+    RepeatSettings,
+)
+from openff.toolkit.topology import Molecule as OFFMolecule
+
+from openfe.protocols.openmm_rfe._rfe_utils import compute
+from openfe.protocols.openmm_utils import (
+    system_validation, settings_validation, system_creation
+)
+
+logger = logging.getLogger(__name__)
+
+
+class PlainMDProtocolResult(gufe.ProtocolResult):
+    """Dict-like container for the output of a PlainMDProtocol
+    outputs filenames for the pdb file and trajectory"""
+    def __init__(self, **data):
+        super().__init__(**data)
+        # data is mapping of str(repeat_id): list[protocolunitresults]
+        if any(len(pur_list) > 2 for pur_list in self.data.values()):
+            raise NotImplementedError("Can't stitch together results yet")
+
+    def get_estimate(self):
+        """Since no results as output --> returns None
+
+        Returns
+        -------
+        None
+        """
+
+        return None
+
+    def get_uncertainty(self):
+        """Since no results as output --> returns None"""
+
+        return None
+
+    def get_traj_filename(self):
```

Comment:
If possible, could you put a return annotation on this and/or in the docstring so that folks can easily check what they are getting back?

Reply:
Added this, does that look ok?